.gitignore

```diff
@@ -1,6 +1,19 @@
-*.o
+
+# Compiled executable and output files
 genvrp
```

Comment:
If genvrp is created in the build directory, then it does not need a specific rule anymore

Reply:
Sure, I'll remove it :)